gradle/validation/error-prone.gradle

```diff
@@ -104,9 +105,9 @@ allprojects { prj ->
             '-Xep:MissingSummary:OFF',
             '-Xep:MixedMutabilityReturnType:OFF',
             '-Xep:ModifiedButNotUsed:OFF',
-            '-Xep:MutableConstantField:OFF',
             '-Xep:MutablePublicArray:OFF',
             '-Xep:NarrowingCompoundAssignment:OFF',
+            '-Xep:NegativeCharLiteral:OFF',
```

Comment:
Solr uses negative chars in a few places correctly and so this check wasn't worth it.